rpcclient/README.md

```diff
@@ -18,7 +18,7 @@ implement and the API is not stable yet.
 
 ## Documentation
 
-* [API Reference](http://godoc.org/github.com/btcsuite/rpcclient)
+* [API Reference](http://godoc.org/github.com/btcsuite/btcrpcclient)
```

Comment:
This moved into the main repository now, so it should be http://godoc.org/github.com/btcsuite/btcd/rpcclient.